src/docs/GetStarted/Developers/configureStylelint/index.mdx

```diff
@@ -61,6 +61,25 @@ Please checkout [Mozaic configuration](/GetStarted/Developers/Configuration/#sty
 
 </Highlight>
 
+### Add your own stylelint rules 
+
+It can appear that sometimes you need to use custom rules in stylelint configuration file for your project, you can add it this way in `stylelint.config.js`: 
+
+```js
+'use strict';
+const mozaicRules = require('@mozaic-ds/css-dev-tools/styleLintConfig.js');
+
+const customRules = {
+  'color-no-invalid-hex': null,
+  'font-family-no-duplicate-names': null
+};
+
+const stylelintConfig = mozaicRules;
+stylelintConfig.rules = {...mozaicRules.rules, ...customRules};
+
+module.exports = stylelintConfig;
+```
+
 
```

Comment:
I think you should also add a "warning" highlight to this section (just below the title), to indicate that this tip is for advanced users only. Something like this: 

```MDX
<Highlight theme="warning" title="Only for advanced users">
The trick below is for advanced users only.
By doing this evolution you go out of the rules set and planned by Mozaic. 
As a result, we cannot fully guarantee the optimal functioning of Mozaic. So be sure of what you are doing.
</Highlight>
````